scripts/deploySidechain.ts

```diff
@@ -73,12 +73,15 @@ export async function deployCanonicalPhase(
         hre,
         new L1Coordinator__factory(deployer),
         "L1Coordinator",
-        [config.lzEndpoint, phase6.booster.address, config.token, phase2.cvx.address, auraProxyOFT.address],
+        [config.l1LzEndpoint, phase6.booster.address, config.token, phase2.cvx.address, auraProxyOFT.address],
         {},
         debug,
         waitForBlocks,
     );
 
+    // const tx = await l1Coordinator.transferOwnership(multisigs.daoMultisig);
```

Comment:
@phijfry  transfer ownship to avoid forgetting doing that in the future.

Reply:
Yes that is fine.

Reply:
Just make sure tests still work